consensus/actor_state.go

```diff
@@ -0,0 +1,96 @@
+package consensus
```

Comment:
nit: I'd be slightly more inclined to call this file `queryer` as the actor state abstraction seems less essential than the `queryer` abstraction.

Reply:
I'm about 49% with you. What holds me back is that `ActorState` is the thing you actually construct and seems a more natural entry point. Breaking this up into multiple files might solve the problem, but I'd like to pollute the package directory as little as possible.